kotlin/android/app/build.gradle

```diff
@@ -32,10 +32,10 @@ android {
 
             // Debug Config
             manifestPlaceholders.hostName = "app.firez.one"
-            buildConfigField("String", "AUTH_HOST", "\"localhost\"")
-            buildConfigField("String", "AUTH_SCHEME", "\"http\"")
-            buildConfigField("Integer", "AUTH_PORT", "8080")
-            buildConfigField("String", "CONTROL_PLANE_URL", "\"ws://localhost:8081/\"")
+            buildConfigField("String", "AUTH_HOST", "\"app.firez.one\"")
+            buildConfigField("String", "AUTH_SCHEME", "\"https\"")
+            buildConfigField("Integer", "AUTH_PORT", "443")
+            buildConfigField("String", "CONTROL_PLANE_URL", "\"wss://app.firez.one/\"")
```

Comment:
@pratikvelani You should now have an account on staging you can use to sign in with the magic link method, which has real Gateways, and a live connlib working with it. Use `pratik@spark.dev` to sign in.

With this change you no longer need to run `PortalMock`.